.github/workflows/_linux-binary-build.yml

```diff
@@ -0,0 +1,237 @@
+name: linux-binary-build
+
+on:
+  workflow_call:
+    inputs:
+      build_name:
+        required: true
+        type: string
+        description: The build's name
+      build_environment:
+        required: true
+        type: string
+        description: The build environment
+      PYTORCH_ROOT:
+        required: true
+        type: string
+        description: Root directory for the pytorch/pytorch repository
+      BUILDER_ROOT:
+        required: true
+        type: string
+        description: Root directory for the pytorch/builder repository
```

Comment:
Just curious, why not specify defaults here?

Reply:
Fair question. With the current usage, there's really no point in having an input with a default value: Either the value is always set or we can hardcode it in.  Default values will mainly become useful once start using this reusable workflow in new scenarios, and it's not guaranteed (from my perspective at least) that these default values would make sense at that point (if they would, then we could again just hard code them instead). 

For now I wanted to pass all the values in using a similar pattern to also make comparing the before/after modes simpler.  This is easy enough to change later if the need arises.